src/site/content/en/lighthouse-seo/canonical/index.md

```diff
@@ -1,105 +1,113 @@
-Canonical links let you explicitly specify which version to crawl.
-Lighthouse flags pages with an invalid `rel=canonical`:
+When multiple pages have similar content, search engines consider them duplicate
+versions of the same page. For example, desktop and mobile versions of a product
+page are often considered duplicates.
+
+Search engines select one of the pages as the _canonical_, or primary, version
+and **crawl** that one more. Valid canonical links let you tell search engines
+which version of a page to crawl and display to users in search results.
+
+{% Aside 'key-term' %}
+_Crawling_ is how a search engine updates its index of content on the web.
+{% endAside %}
+
+Using canonical links has many advantages:
+- It helps search engines consolidate multiple URLs into a single, preferred URL. For example,
+  if other sites put query parameters on the ends of links to your page, search engines
+  consolidate those URLs to your preferred version.
+- It simplifies tracking methods. Tracking one URL is easier than tracking many.
+- It improves the page ranking of syndicated content by consolidating the syndicated links to
+  your original content back to your preferred URL.
+
+## How the Lighthouse audit for canonical links fails
+
+Lighthouse flags any page with an invalid canonical link:
 
 <figure class="w-figure">
-  <img class="w-screenshot w-screenshot--filled" src="canonical.png" alt="Lighthouse audit showing document with invalid canonical link">
-  <figcaption class="w-figcaption">
-    Document has an invalid canonical link.
-  </figcaption>
+  <img class="w-screenshot w-screenshot" src="canonical.png" alt="Lighthouse audit showing document with invalid canonical link">
 </figure>
 
-## What causes the audit to fail
-
-Lighthouse marks the audit as a fail if any of the following are true:
+A page fails this audit if any of the following conditions are met:
 
-- There is more than 1 canonical link.
+- There is more than one canonical link.
 - The canonical link is not a valid URL.
 - The canonical link points to a page for a different region or language.
 - The canonical link points to a different domain.
-- The canonical link points to the site root. Note that this may be valid in some scenarios,
-  such as for AMP or mobile page variations, but nonetheless Lighthouse marks this scenario as
-  a failure.
+- The canonical link points to the site root. Note that this scenario may be
+  valid in some scenarios, such as for AMP or mobile page variations, but
+  Lighthouse nonetheless treats it as a failure.
 
 {% include 'content/lighthouse-seo/scoring.njk' %}
 
 ## How to add canonical links to your pages
 
-Add a canonical link element to the `head` element of your HTML:
+There are two options for specifying a canonical link.
 
-```html
+**Option 1:** Add a `<link rel=canonical>` element to the `<head>` of the page:
+
+```html/4
 <!doctype html>
-<html>
+<html lang="en">
   <head>
-    ...
+    …
```

Comment:
I don't like this because I don't have ready access to this character. It's easier to just use `...`

Reply:
I won't die on this hill but my rule of thumb is to just reduce friction in the review process unless absolutely necessary, in this case using a special character instead of `...` is needless friction

Reply:
We've already added that rule to the content linting and provided the ellipsis character in the warning to reduce friction as much as possible, but suppose we could take them out. @robdodson, what do you think?